lib/PuppeteerSharp/ExecutionContext.cs

```diff
@@ -22,44 +24,86 @@ public ExecutionContext(Session client, ContextPayload contextPayload, Func<dyna
         public string FrameId { get; internal set; }
         public bool IsDefault { get; internal set; }
 
-        public async Task<dynamic> Evaluate(string pageFunction, params object[] args)
+        public async Task<dynamic> EvaluateAsync(string script, params object[] args)
         {
-            var handle = await EvaluateHandleAsync(pageFunction, args);
+            var handle = await EvaluateHandleAsync(script, args);
             dynamic result = await handle.JsonValue();
```

Comment:
is there a case when `result` isn't a `JToken`? we might not need this overload

Reply:
The problem is that we can't use `dynamic` when we call `EvaluateAsync<T>`. So when we need a `dynamic` we should call this function.

Thoughts?

Reply:
when would want our response to be of type `dynamic`? we always know what we expect the method to return

Reply:
Here, for instance, https://github.com/kblok/puppeteer-sharp/pull/94/files#diff-a700c62632ba5e31465dc34bc7e2d89dR72

I wouldn't like to have to create a class to access `result.foo`

Reply:
well, that's because you sent a dynamic object in the first place and returned it... anyhow, it doesn't hurt